File: 백준/Bronze/25304. 영수증/영수증.cc
#include <stdio.h>

int main(void) {
    int x, n, sumPrice = 0;
    scanf("%d", &x);
    scanf("%d", &n);

    while(n--) {
        int a, b;
        scanf("%d %d", &a, &b);
        sumPrice += a * b;
    }

    if (sumPrice == x) printf("Yes");
    else printf("No");

    return 0;
}

File: 백준/Bronze/25304. 영수증/영수증.py
import sys
input = sys.stdin.readline

x = int(input().rstrip())
n = int(input().rstrip())
sumPrice = 0

for _ in range(n):
    a, b = map(int, input().rstrip().split())
    sumPrice += a * b

if sumPrice == x: print('Yes')
else: print('No')